### PR TITLE
Add eva command to check batman status

### PR DESCRIPTION
**Pull request by vivekiitkgp**
Signed-off-by: Vivek Rai <vivekrai.iitkgp@gmail.com>

### CODE REVIEW

File: scripts/batman.coffee [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{spawn} = require 'child_process'
+
+batman_happy = [
+  'batman is up and running!',
+  'batman never sleeps!',
+  'batman is monitoring MetaKGP from the shadows!'
+]
+
+batman_sad = [
+  'batman needs its maintainer!',
```
Comment (icyflame) [inline scripts/batman.coffee]: @vivekiitkgp This line would be a lot more funnier if it read `batman needs alfred to bring him some tea` :smile:

**Comment by vivekiitkgp** [comment]
I had written that initially, but thought it'll reveal the cheesy side of
me. :P

--
https://raivivek.in

**Comment by amrav** [comment]
How does this work? Is eva running on the server now?

**Comment by icyflame** [comment]
@amrav No no, Eva is still on Heroku. @vivekiitkgp had a slight confusion about how to add some functionality that we were discussing (Eg: `eva is xtinct logged in?` or `eva check ssh status` or `eva check mysql status`, etc)

It's all sorted now.

**Pull request closed without merging**